### ISSUE TITLE
Some text elements are inconsistent between editor, preview, and publish in RTL mode

### ISSUE THREAD

**Issue by zachhale** [GoogleForCreators/web-stories-wp#6068]
## Bug Description

During investigation of RTL support for Page Layouts/Templates in #5909, it was found that some text elements render differently in both previews and the editor when in RTL mode than they do when published/previewed. 

Pages created in RTL or LTR mode both look like the LTR when published, but differ in the editor/page previews.

## Expected Behaviour

LTR and RTL should not change how elements appear in the editor. Pages created from templates/page layouts in RTL mode should render the same when previewed/published than when they are edited.

## Steps to Reproduce

1. Switch to RTL mode
2. Add a page layout that has an issue (Ex: Cooking List "Top 8" page)
3. Text elements render right aligned but should be left aligned
4. Publish/Preview. The elements are left aligned as they should appear in the editor.

## Screenshots

Editor content of the same exact same story page (from template/page layout):
| Editor RTL | Editor LTR |
|---|---|
| ![image](https://user-images.githubusercontent.com/3869/105907900-61029300-5fda-11eb-80a0-a8d201e08bd7.png) | ![image](https://user-images.githubusercontent.com/3869/105907920-6a8bfb00-5fda-11eb-8b31-5f390f7c8ce8.png) |

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
cc @miina @barklund
